backend/tests/test_search.py

```diff
@@ -128,3 +128,29 @@ def test_limit(client, db):
     response = json.loads(rv.data)
     assert len(response) == limit
     assert response[0]['upc'] == matches[0].upc
+
+
+def test_search_close(client, db):
```

Comment:
Please move these tests to their own file, test_gps_search.py

Reply:
Done